src/hashstore/filehashstore.py

```diff
@@ -1021,80 +1460,241 @@ def _mktmpmetadata(self, stream):
+                    "FileHashStore - _validate_arg_data: Data string cannot be empty."
                 )
                 logging.error(exception_string)
                 raise TypeError(exception_string)
+        return True
 
-    def _validate_algorithms_and_checksum(
+    def _check_arg_algorithms_and_checksum(
         self, additional_algorithm, checksum, checksum_algorithm
     ):
-        """Determines whether calling app has supplied the necessary arguments to validate
-        an object with a checksum value
-
-        Args:
-            additional_algorithm: value of additional algorithm to calculate
-            checksum (string): value of checksum
-            checksum_algorithm (string): algorithm of checksum
+        """Determines whether the caller has supplied the necessary arguments to validate
+        an object with a checksum value.
+
+        :param additional_algorithm: Value of the additional algorithm to calculate.
+        :type additional_algorithm: str or None
+        :param checksum: Value of the checksum.
+        :type checksum: str or None
+        :param checksum_algorithm: Algorithm of the checksum.
+        :type checksum_algorithm: str or None
+
+        :return: Hashlib-compatible string or 'None' for additional_algorithm and
+            checksum_algorithm.
+        :rtype: str
         """
         additional_algorithm_checked = None
         if additional_algorithm != self.algorithm and additional_algorithm is not None:
             # Set additional_algorithm
             additional_algorithm_checked = self.clean_algorithm(additional_algorithm)
         checksum_algorithm_checked = None
         if checksum is not None:
-            self._is_string_none_or_empty(
+            self._check_string(
                 checksum_algorithm,
                 "checksum_algorithm",
-                "validate_checksum_args (store_object)",
+                "_check_arg_algorithms_and_checksum (store_object)",
             )
         if checksum_algorithm is not None:
-            self._is_string_none_or_empty(
+            self._check_string(
                 checksum,
                 "checksum",
-                "validate_checksum_args (store_object)",
+                "_check_arg_algorithms_and_checksum (store_object)",
             )
             # Set checksum_algorithm
             checksum_algorithm_checked = self.clean_algorithm(checksum_algorithm)
         return additional_algorithm_checked, checksum_algorithm_checked
 
+    def _check_arg_format_id(self, format_id, method):
+        """Determines the metadata namespace (format_id) to use for storing,
+        retrieving, and deleting metadata.
+
+        :param str format_id: Metadata namespace to review.
+        :param str method: Calling method for logging purposes.
+
+        :return: Valid metadata namespace.
+        :rtype: str
+        """
+        checked_format_id = None
+        if format_id is not None and format_id.replace(" ", "") == "":
```

Comment:
does this work as a more concise/readable option?

```python
if format_id and not format_id.strip():
```

Reply:
Beautiful.... thank you!